k8s/upgrades/0.6.0-0.7.0/oebs_update.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash -x
+
+################################################################
+# STEP: Get Persistent Volume (PV) name as argument            #
+#                                                              #
+# NOTES: Obtain the pv to upgrade via "kubectl get pv"         #
+################################################################
+
+function usage() {
+    echo 
+    echo "Usage:"
+    echo 
+    echo "$0 <pv-name> <replica-label>"
```

Comment:
Is it replica-label or node-label ?

Reply:
good point. it is actually a custom label that user need to provide.